README.md

```diff
@@ -4,63 +4,117 @@
 
 ## Description
 
-This repository contains two workflows for metagenome and metatranscriptome assembly of short read data. MetaSPAdes is used as default for paired end data, and MEGAHIT for single end data. MEGAHIT can be specified as the default assembler in the yaml file if preferred. Steps include:
+This repository contains two workflows for metagenome and metatranscriptome assembly of short read data. MetaSPAdes is used as default for paired-end data, and MEGAHIT for single-end data and co-assemblies. MEGAHIT can be specified as the default assembler in the yaml file if preferred. Steps include:
 
-QC - removal of short reads, low quality regions, adapters and host decontamination
+  * _QC_: removal of short reads, low quality regions, adapters and host decontamination
+  * _Assembly_: with metaSPADES or MEGAHIT
+  * _Post-assembly_: Host and PhiX decontamination, contig length filter (500bp), stats generation
 
-Assembly - with metaSPADES or MEGAHIT
+## Requirements - How to install
 
-Post-assembly - Host and PhiX decontamination, contig length filter (500bp), stats generation.
+This pipeline requires a conda environment with cwltool, blastn, and metaspades. If created with `requirements.yml`, the environment will be called `cwl_assembly`. 
 
-Multiple input read files can also be specified for co-assembly.
-
-## Requirements
-
-This pipeline requires and environment with cwltool, blastn, metaspades and megahit.
+```
+conda env create -f requirements.yml
+conda activate cwl_assembly
+pip install cwltool
+```
 
 ## Databases
 
-Predownload fasta files for host decontamination and generate:
-    - bwa index folder
-    - blast index folder
+You will need to pre-download fasta files for host decontamination and generate the following databases accordingly:
+  * bwa index
+  * blast index
     
 Specify the locations in the yaml file when running the pipeline.
 
-
 ## Main pipeline executables
 
-src/workflows/metagenome_pipeline.cwl
-src/workflows/metatranscriptome_pipeline.cwl
+  * `src/workflows/metagenome_pipeline.cwl`
+  * `src/workflows/metatranscriptome_pipeline.cwl`
+
+## Example command
+
+```cwltool --singularity --outdir ${OUTDIR} ${CWL} ${YML}```
+
+`$CWL` is going to be one of the executables mentioned above
+`$YML` should be a config yaml file including entries among what follows. In the case of single-end assemblies, choose parameters `reads1` (and `reads2` in the case of paired-end). For co-assemblies, please use `multiple_reads_1` for multiple single-end samples, jointly with `multiple_reads_2` for multiple paired-end combinations. 
+
+```
```

Comment:
This would be ideal in a file example_run.yaml for people to refer to. Can be downloaded/cloned and edited then.

Reply:
Cut and paste it into a yaml and then just refer to that in the text would be my suggestion. But your preference :)